core/src/main/java/com/scalar/db/api/Put.java

```diff
@@ -26,7 +27,7 @@
  */
 @NotThreadSafe
 public class Put extends Mutation {
-  private final Map<String, Value<?>> values;
+  private final Map<String, Optional<Value<?>>> values;
```

Comment:
To express NULL values, I changed the type of this map value to `Optional<Value<?>>`. `Optional.empty()` means a NULL value.